### PR TITLE
Support photo diary images better — treat them as main article images

### PR DESCRIPTION
**Pull request by matthew-andrews**
cc @ironsidevsquincy

### CODE REVIEW

**Comment by matthew-andrews** [comment]
screen shot 2015-05-19 at 17 14 24

File: server/controllers/capi.js [hunk 1]
```diff
@@ -90,7 +90,8 @@ module.exports = function(req, res, next) {
 						isColumnist: isColumnist,
 						// if there's a main image, or slideshow or video, we overlap them on the header
 						headerOverlap:
-							$.root().children('.article__main-image, ft-slideshow:first-child, .article__video-wrapper:first-child').length,
+							$.root().children('.article__main-image, ft-slideshow:first-child, .article__video-wrapper:first-child').length
+							|| $.root().first().children('.article__main-image'),
```
Comment (ironsidevsquincy) [inline server/controllers/capi.js]: combine with above? `*:first-child > .article__main-image`

**Comment by ironsidevsquincy** [comment]
:+1:

File: server/controllers/capi.js (hunk 1 above)
Comment (matthew-andrews) [inline server/controllers/capi.js]: hmm it didn't work with `a > .article__main-image` presumably because using `children` rather than `find`

**Pull request merged**